### ISSUE TITLE
Some values are null on spawn an Actor on OnAwake

### ISSUE THREAD

**Issue by RuanLucasGD** [FlaxEngine/FlaxEngine#1367]
**Issue description:**
If you spawn an Actor on OnAwake using `PrefabManager.SpawnPrefab`, the Transform and Parent of spawned actor is null.

**Steps to reproduce:**
Spawn a prefab on OnAwake.

Example Code:

```
        public Prefab prefab;

        /// <inheritdoc/>
        public override void OnAwake()
        {
            PrefabManager.SpawnPrefab(prefab, Actor.Position, Actor.Orientation);
        }
```

**Flax version:**
1.6

**Comment by Withaust** [comment]
I believe @NoriteSC was working on a fix regarding this problem.

**Comment by mafiesto4** [comment]
Yeah we gonna need to set objects transform cache (in hierarchy, from local transforms) before calling any objects initialization (eg. `OnAwake`)

**Comment by NoriteSC** [comment]
[mafiesto4](https://github.com/mafiesto4) is a fix #1337

**Comment by mafiesto4** [comment]
Fixed in #1337
